### PR TITLE
WT-9536 Fix alphabetical ordering of flags in flags.py

### PR DESCRIPTION
**Pull request by lting0804**
- Caused by an inconsistency between tabs and spaces
- Further Investigation shows that tabbing in a header file causes the tab to be automatically converted into a space hence this problem does not exist in any other header file upon running `flags.py`
- Occurs in wiredtiger.h since it is built based on wiredtiger.in and .in files keep the `\t` character rather than replacing it with spaces
- The following output.txt shows the result of spaces and tabs upon running the `flags.py` with the following addition listed below:
[output.txt](https://github.com/wiredtiger/wiredtiger/files/10191923/output.txt)

- The following addition to the code is added when the stop header is found
```
print(defines)
for d in defines: 
    if d[d.find('#define') + len('#define')] == '\t': # Checks if the next character after #define is a tab
            tab_found = 1
    if d[d.find('#define') + len('#define')] == ' ':
            space_found = 1
    if tab_found and space_found: 
        print("tabs and spaces in " + name)
    elif tab_found: 
        print("tabs in " + name)
    elif space_found: 
        print("spaces in " + name)
    if tab_found == 0: 
        print("no tabs in " + name)
    if space_found == 0: 
        print("no spaces in " + name)
```

### CODE REVIEW

File: src/include/wiredtiger.in [hunk 1]
```diff
@@ -739,40 +739,40 @@ struct __wt_cursor {
 	WT_ITEM lower_bound, upper_bound;
 
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define	WT_CURSTD_APPEND		0x000000001ull
-#define	WT_CURSTD_BULK			0x000000002ull
```
Comment (jiechenbo) [inline src/include/wiredtiger.in:743]: This flag seems unaligned from the rest, is that automatically generated?

Comment (lting0804) [reply src/include/wiredtiger.in:743]: only the flag is automatically generated - as we are only concerned about the character after the #define as it is integral to sorting the rest was not touched

**Review by Siddhartha8899 — APPROVED** [review]
Excellent work! LGTM!

**Review by jiechenbo — APPROVED** [review]
LGTM! nice work 👍

**Pull request merged**